api/src/paths/project/{projectId}/survey/{surveyId}/observation/submission/upload.ts

```diff
@@ -199,24 +199,20 @@ export const insertSurveyOccurrenceSubmission = async (
   source: string,
   inputFileName: string,
   connection: IDBConnection
-): Promise<any> => {
-  const insertSqlStatement = queries.survey.insertSurveyOccurrenceSubmissionSQL({
+): Promise<{ submissionId: number }> => {
```

Comment:
Is there any reason to keep these functions in this endpoint? Could we move them to a service file?

Reply:
ie `insertSurveyOccurrenceSubmission`, `updateSurveyOccurrenceSubmissionWithKey`

Reply:
That's a fair question. We could certainly inline these functions - I don't think there's a strong case to have `insertSurveyOccurrenceSubmission` and `updateSurveyOccurrenceSubmissionWithKey` exist as separate functions, if only to make testing the endpoint simpler.

Granted, I think the reason that we don't call these two steps inside another "upload occurrence" service file method is because of the file upload to S3; We first need to insert the occurrence record in order to generate an S3 key, both to update the record and to upload the file. Consequently I don't think moving this logic to a service method provides much value because it would only be called by this endpoint.